src/miniwiki.rs

```diff
@@ -58,9 +59,15 @@ fn render_class(class: &DumpClass) -> HtmlContent {
         .map(String::as_str)
         .unwrap_or(DEFAULT_DESCRIPTION);
 
+    let mut element_class = "dump-class".to_owned();
+    if class.tags.contains(&"Deprecated".to_owned())

```

Comment:
Should definitely be able to just use `contains` here. If not, I'm going insane!

```suggestion
    if class.tags.contains("Deprecated")
```

`BTreeSet`'s `contains` implementation should work for any type that can act as a borrowed version of the stored value.

See: https://doc.rust-lang.org/std/collections/struct.BTreeSet.html#method.contains